static-assets/components/cstudio-dialogs/select-content-type.js

```diff
@@ -314,16 +314,7 @@ CStudioAuthoring.Dialogs.DialogSelectContentType = {
      * EMO-8604, calling closeDialog to remove pop up from DOM.
      */
     CStudioAuthoring.Dialogs.DialogSelectContentType.closeDialog();
-    if (CStudioAuthoring.Dialogs.DialogSelectContentType.changeTemplateCalled == true) {
-      CStudioAuthoring.Service.changeContentType(
-        CStudioAuthoringContext.site,
-        oCurrentTextNode.data.uri,
-        selectedType,
-        args.self.onSaveCallback
-      );
-    } else {
-      args.self.onSaveCallback.success(selectedType);
-    }
```

Comment:
What's up with this deletion?

Reply:
change content type is now done using the new react component, so this validation is not required anymore